presto-benchmark-runner/pom.xml

```diff
@@ -177,6 +183,12 @@
             <groupId>com.facebook.presto</groupId>
             <artifactId>testing-mysql-server-5</artifactId>
             <scope>test</scope>
+            <exclusions>
```

Comment:
Does this mean the tests are still running with MySQL server version 5?

Reply:
The scope of this dependency is set to test, indicating that it's only for the test environment. answering to ur question: yes

Reply:
I think we need a new version of this dependency for version 8 so our tests and prod are consistent.

Reply:
I was looking for the same but could not find other similar dependency.

Reply:
We'll need to create it, the source code is here: https://github.com/prestodb/testing-mysql-server

Reply:
cloned the source repo and was trying to build 
![Uploading Screenshot 2024-01-03 at 9.55.26 AM.png…]()
biut facing this even after installing the required libraries, Can someone help with this?